impl/src/main/java/org/ehcache/impl/persistence/DefaultDiskResourceService.java

```diff
@@ -113,7 +121,7 @@ public void releasePersistenceSpaceIdentifier(PersistenceSpaceIdentifier<?> iden
       }
     }
     if (name == null) {
-      throw new CachePersistenceException("Unknown space " + identifier);
+      throw newCachePersistenceException(identifier);
```

Comment:
Hum... You are right that some refactoring can be done. It is not related to my code so I prefer to merge this PR and do it after